regulations/templates/regulations/partials/node_types/Paragraph.html

```diff
@@ -4,7 +4,7 @@
 <p tabindex="-1" id="{{node.label | join:'-'}}" class="reg-section depth-{{node|pdepth}}">
     {% if node.label and node.label|length > 0 %}
         {% paragraph_formatter cfr_title node.label as formatted_citation %}
-        {% include "regulations/partials/copy-btn.html" with title=node.label btn_type="icon" formatted_citation=formatted_citation %}
+        <copy-btn btn_type="icon" title="{{nobe.label | join:'.'}}" hash="{{title | join:'-'}}" formatted_citation="{{formatted_citation}}"></copy-btn>
```

Comment:
node.label?

Reply:
yes, sometimes my fingers have a mind of their own

Reply:
and I'm actually glad you caught the typo because I was passing the wrong hash variable as well.